src/objc-renderer.ts

```diff
@@ -243,14 +243,77 @@ function enumerationIsPublic(isPublic:boolean):(enumeration:ObjC.Enumeration) =>
   };
 }
 
+const NS_ASSUME_NONNULL_BEGIN:string = 'NS_ASSUME_NONNULL_BEGIN';
+const NS_ASSUME_NONNULL_END:string = 'NS_ASSUME_NONNULL_END';
+
+function prefixClassMacros(classInfo:ObjC.Class):string[] {
+  var macros:string[] = [];
+  switch (classInfo.nullability) {
+    case ObjC.ClassNullability.assumeNonnull:
+      macros.push(NS_ASSUME_NONNULL_BEGIN);
```

Comment:
We shouldn't be mutating the array from inside the if. For now, I would simply return the array from here. In the future when we add more, we can figure out the right way to build the array up